### ISSUE TITLE
Is there a way to login automatically (without password)?

### ISSUE THREAD

**Issue by fmv1992** [fairyglade/ly#266]
Hi, thanks for creating this software.

Is there a way to login automatically (without password)? I can't find this in the config file or documentation/google.

**Comment by skittlesvampir** [comment]
Hello, Mr. Vieira,
I‘m not sure if I understood your question right, but if you want to disable the password for your user, run the following command:
```passwd -d felipe```
(see [https://www.cyberciti.biz/faq/linux-delete-user-password/](https://www.cyberciti.biz/faq/linux-delete-user-password/) for more information about disabling the password).
Of course you need to replace felipe with your username.
Sincerery,
Skittlesvampir.

**Comment by Wael2727** [comment]
no, what he means is that you put an option in the config file where it automatically takes "/etc/ly/save" and imports it into the login screen, it means that it automatically login's your user and wm depending on what is in the file, it's already there but it has no purpose
so try to just use that instead

**Comment by fmv1992** [comment]
Hi, thanks for your replies.

> no, what he means is that you put an option in the config file where it automatically takes "/etc/ly/save" and imports it into the login screen, it means that it automatically login's your user and wm depending on what is in the file, it's already there but it has no purpose

Yes. Ideally the password should not need to be stored in plaintext. Other display managers (`lightdm` in Lubuntu for instance) allow "passwordless" start. The real life scenario is: I already have full disk encryption, so it makes no sense to input my password twice.

Thanks for taking this in consideration.

**Comment by manel-bc** [comment]
Not fully automatic, but you can already enable logging in without a password for a given user.
Edit `/etc/pam.d/ly` to include `auth       sufficient   pam_succeed_if.so user ingroup emptypasslogin` at the top of the stack/list:
```
#%PAM-1.0

auth       sufficient   pam_succeed_if.so user ingroup emptypasslogin
auth       include      login
.
.
.
```
Any user in group `emptypasslogin`, or whichever group you chose, will be able to login without entering their password.

**Comment by wael444** [comment]
that is extremely unsafe.

**Comment by manel-bc** [comment]
> that is extremely unsafe.

Care to elaborate?

**Comment by wael444** [comment]
no

**Comment by Estebiu** [comment]
> 

chad

**Comment by RodrigoPrestes** [comment]
> Care to elaborate?

Passwordless would not be secure because if I log out other people can get to my user without typing the password, in this encrypted disk scenario the only way to be secure would be to turn off the computer -which is not very practical.

> Yes. Ideally the password should not need to be stored in plaintext. Other display managers (`lightdm` in Lubuntu for instance) allow "passwordless" start. The real life scenario is: I already have full disk encryption, so it makes no sense to input my password twice.

I believe what they had in mind is to not store the password in a file, look at this example of how GDM does it, it's just a config file.
https://wiki.archlinux.org/title/GDM#Automatic_login

And still on the GDM example, passwordless login is a different thing: https://wiki.archlinux.org/title/GDM#Passwordless_login
